packages/website/docs/.vitepress/config.ts

```diff
@@ -207,6 +232,75 @@ function getHeadTags(env: NodeJS.ProcessEnv): HeadConfig[] {
   return tags;
 }
 
+function transformHead({ pageData }: TransformContext): HeadConfig[] {
+  const head: HeadConfig[] = [];
+
+  head.push([
+    "meta",
+    {
+      property: "og:url",
+      content: pageData.frontmatter.path
+        ? `${METADATA_DEFAULT.baseUrl}${pageData.frontmatter.path}`
+        : METADATA_DEFAULT.baseUrl,
+    },
+  ]);
+  head.push([
+    "meta",
+    {
+      property: "og:title",
+      content:
+        `${METADATA_DEFAULT.title} - ${pageData.frontmatter.title}` ||
+        METADATA_DEFAULT.title,
+    },
+  ]);
+  head.push([
+    "meta",
+    {
+      property: "og:description",
+      content: pageData.frontmatter.description || METADATA_DEFAULT.description,
+    },
+  ]);
+  head.push([
+    "meta",
+    {
+      property: "og:image",
+      content: pageData.frontmatter.imageTitle
+        ? `${METADATA_DEFAULT.baseUrl}/api/og?title=${encodeURIComponent(
+            pageData.frontmatter.imageTitle
+          )}`
+        : METADATA_DEFAULT.image,
+    },
+  ]);
+
+  head.push([
+    "meta",
+    {
+      property: "twitter:title",
+      content:
+        `${METADATA_DEFAULT.title} - ${pageData.frontmatter.title}` ||
```

Comment:
I don't think this syntax will work? if `frontmatter.title` is undefined, I think it will just render `BlockNote - undefined` right?

(there's a couple more occurences in this file)